blueprints/banking_blueprint/__init__.py

```diff
@@ -0,0 +1,474 @@
+    replies = [
+        "You can activate your card by logging clicking on the link sent to your email"
+        " at {email} or you can call us at 1-800-432-3117."
+    ]
+    responder.reply(replies)
+
+
+@app.handle(default=True)
+def default(request, responder):
+    replies = [
+        "I'm not sure how to help with that. Try asking about account"
+        " balances or ordering checks",
+        "I'm afraid I don't know how to help with that."
+        " You can try to ask to check your balance or pay a bill",
+        "Sorry, I do not know what that is. Try asking about"
+        " card activation or reporting a stolen card",
+        "I'm afraid I can not help with that. A few banking tasks"
+        " you can try are, transferring "
+        "balances and paying bills",
+    ]
+    responder.reply(replies)
+
+
+@app.handle(intent="credit_due_date")
+def check_due_date_handler(request, responder):
+    replies = [
+        "Your credit card bill is due on the 15th of every month,"
+        " late payments will result in a $25 fee."
+        " Say 'pay my credit card bill' to pay the due balance."
+    ]
+    responder.reply(replies)
+
+
+@app.auto_fill(intent="transfer_balances", form=entity_form)
+def transfer_balances_handler(request, responder):
+    if not user_data:
+        _pull_data(request)
+    for entity in request.entities:
+        if entity["type"] == "account_type":
+            if _credit_check(request, responder, entity):
+                return
+            if entity["role"] == "origin":
+                responder.slots["origin"] = (
+                    entity["value"][0]["cname"]
+                    if len(entity["value"]) > 0
+                    else entity["text"]
+                )
+            elif entity["role"] == "dest":
+                responder.slots["dest"] = (
+                    entity["value"][0]["cname"]
+                    if len(entity["value"]) > 0
+                    else entity["text"]
+                )
+        else:
+            responder.slots["amount"] = (
+                entity["value"][0]["value"]
+                if len(entity["value"]) > 0
+                else entity["text"]
+            )
+    if _check_value(responder):
+        responder.reply(
+            [
+                "All right. A transfer of $"
+                + format(responder.slots["amount"], ".2f")
+                + " dollars from your "
+                "{origin} to your {dest} has been intiated."
+            ]
+        )
+        _put(
+            responder.slots["origin"],
+            _get(responder.slots["origin"]) - responder.slots["amount"],
+        )
+        _put(
+            responder.slots["dest"],
+            _get(responder.slots["dest"]) + responder.slots["amount"],
+        )
+    else:
+        responder.reply(
+            [
+                "You do not have $"
+                + format(responder.slots["amount"], ".2f")
+                + " in your {origin} account. "
```

Comment:
Are we still supporting python 3.5? I would prefer f-strings if not, and if so would rather do
` "You do not have {amount:.2f} in your {{origin}} acount".format(responder.slots["amount"])`